pallets/pallet-did-lookup/src/lib.rs

```diff
@@ -327,6 +369,48 @@ pub mod pallet {
 
 			LinkableAccountDepositCollector::<T>::update_deposit(&account)
 		}
+
+		// TODO: Docs, weights
+		#[pallet::weight(T::DbWeight::get().reads_writes(2, 4))]
+		pub fn migrate_account_id(origin: OriginFor<T>, account_id: AccountIdOf<T>) -> DispatchResult {
+			ensure_signed(origin)?;
+			// TODO: Do we need to ensure whether MigrationOngoing? Shouldn't due to
+			// CallFilter.
+
+			let linkable_account_id: LinkableAccountId = account_id.clone().into();
+			let did_id = crate::migrations::do_migrate_account_id::<T>(account_id, linkable_account_id.clone())
+				.ok_or(Error::<T>::AlreadyMigrated)?;
+
+			Self::deposit_event(Event::<T>::Migrated {
+				account_id: linkable_account_id,
+				did_id,
+			});
+
+			Ok(())
+		}
+
+		// TODO: Docs, weights
+		#[pallet::weight(T::DbWeight::get().reads_writes(1, 2))]
+		pub fn verify_migration(origin: OriginFor<T>) -> DispatchResult {
```

Comment:
I know it's just temporary, but for future reference, can we rename this to something like `try_finalize_migration` or `try_end_migration`?